src/main/java/org/primefaces/convert/CombinedDateTimePatternTokenVisitor.java

```diff
@@ -99,8 +99,6 @@ public void visitTokenLetter(int letter, int repetitions) {
 
             // time zone
             case 'z':
-                visitZonezToken(repetitions, builder);
-                break;
             case 'Z':
```

Comment:
Made this a fall through since both 'z' and 'Z' do the same thing.

Reply:
I made a separate method for each case, and for 'z' and 'Z' the JQueryUI pattern happens to be the same for both. Makes the code a tiny bit easier to read imo, but I've got no problem with this change.